configure.ac

```diff
@@ -846,6 +846,13 @@ AC_ARG_ENABLE([wpas-dpp],
     [ ENABLED_WPAS_DPP=no ]
     )
 
+# ntp support
```

Comment:
Can you add NTP to the documentation section at line 774 "# List of open source project defines using our openssl compatibility layer:"?